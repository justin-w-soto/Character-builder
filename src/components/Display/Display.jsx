import React from 'react'

export default function Display ({ catchphrases }) {
    
    return (
        <>
          
            {catchphrases.map((catchphrase) => 
            <p 
            key={catchphrase}>{catchphrase}
            </p>
            )}
            <br></br>
        </>
    );
}
