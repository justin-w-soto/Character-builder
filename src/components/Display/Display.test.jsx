/* eslint-disable no-unused-vars */
import { render, screen } from '@testing-library/react';
import Display from './Display';

it('Displays a catchphrase', () => {
    const { container } = render(
        <Display catchphrases={['hello', 'dude']} />
    )
    expect(container).toMatchSnapshot();
})