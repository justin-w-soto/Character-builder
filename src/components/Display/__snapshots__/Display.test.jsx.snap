// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Displays a catchphrase 1`] = `
<div>
  <p>
    hello
  </p>
  <p>
    dude
  </p>
  <br />
</div>
`;
